Makefile

```diff
@@ -8,16 +8,16 @@ all: synapse.s9pk
 install: synapse.s9pk
 	appmgr install synapse.s9pk
 
-synapse.s9pk: manifest.yaml config_spec.yaml config_rules.yaml image.tar instructions.md
+synapse.s9pk: manifest.yaml config_spec.yaml config_rules.yaml image.tar
 	appmgr -vv pack $(shell pwd) -o synapse.s9pk
```

Comment:
update to `embassy-sdk pack` and `embasy-sdk verify <app-id>.s9pk` - see [here](https://github.com/Start9Labs/cups-wrapper/pull/1/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R22) for an example